lib/express/myinfo.js

```diff
@@ -114,6 +114,8 @@ function config (app, { serviceProvider, port }) {
     const samlArtifact = SAMLart.replace(/ /g, '+')
     const samlArtifactBuffer = Buffer.from(samlArtifact, 'base64')
     const index = samlArtifactBuffer.readInt8(samlArtifactBuffer.length - 1)
+    // use env NRIC as open_login_page = false
+    if (index === -1) index = assertions.identities.singPass.indexOf(singPassNric)
```

Comment:
Would not work given that index is `const`
Nitpick - for clarity, enclose even single-line statements in brackets